modules/appeals_api/config/schemas/10182.json

```diff
@@ -46,18 +46,50 @@
       "additionalProperties": false,
       "properties": {
         "homeless":            { "type": "boolean" },
```

Comment:
Can/does the schema validate homeless OR address with `oneOf`?

Reply:
This was nixed way back when, during the early days of the NOD project, due to the complexity and supposedly unreadable errors. Maybe we should revisit?

Reply:
If we are going to revisit, let's do it as a separate ticket.

Reply:
Potentially! With Drew's error class, this might be a good way to validate the incoming schema.

Reply:
Creating a ticket now